### PR TITLE
remove logs / normalize path

### PR DESCRIPTION
**Pull request by gonpombo8**

### CODE REVIEW

File: test/sdk-commands/utils/data-layer-fs.spec.ts [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+    test('[Win] should return the directory name of the given path', () => {
+      const input = 'scene\\assets\\main.composite'
+      const expectedOutput = 'scene/assets'
+      const dirname = path.dirname
+      path.dirname = path.win32.dirname
+      expect(fsInterface.dirname(input)).toEqual(expectedOutput)
+      path.dirname = dirname
+    })
+
+    test('should return an empty string if the path is a file name', () => {
```
Comment (cazala) [inline test/sdk-commands/utils/data-layer-fs.spec.ts:42]: nip tick here, but it should be a `"."` rather than an empty string no?

File: Makefile [hunk 2]
```diff
@@ -56,7 +56,6 @@ lint-fix: sync-deps
 TESTARGS ?= test/
 test:
 	node_modules/.bin/jest --detectOpenHandles --colors $(TESTARGS)
-	make test-inspector
```
Comment (cazala) [inline Makefile:59]: Why is this removed?

**Review by cazala — APPROVED** [review]

File: test/sdk-commands/utils/data-layer-fs.spec.ts (hunk 1 above)
Comment (gonpombo8) [reply test/sdk-commands/utils/data-layer-fs.spec.ts:42]: yes !

File: Makefile (hunk 2 above)
Comment (gonpombo8) [reply Makefile:59]: cause when you want to run the inspector test you use the `make test-inspector` command.
and when you want tu run the sdk/ecs test you use the `make test`. Just to save time and ignore the inspecrtor tests when you want to run a single test using the TESTARGS

Comment (gonpombo8) [reply Makefile:59]: this is not being used by the github actions.
there we use the make test-cli, make test-inspector and make test-coverage.

Maybe we can rename it 🤔  like `make test-sdk` instead of just test
